lib/include/tree_sitter/parser.h

```diff
@@ -130,9 +130,16 @@ struct TSLanguage {
  *  Lexer Macros
  */
 
+#ifdef _MSC_VER
+#define UNUSED __pragma(warning(suppress : 4101))
+#else
+#define UNUSED __attribute__((unused))
+#endif
+
 #define START_LEXER()           \
   bool result = false;          \
   bool skip = false;            \
+  UNUSED_VAR                    \
```

Comment:
May be `UNUSED` and not `UNUSED_VAR`?

Reply:
yep..originally named it UNUSED_VAR but changed it in the def..